templates/omero-web-config-signup.j2

```diff
@@ -10,4 +10,4 @@ config set -- omero.web.signup.group.name %Y-%m
 config set -- omero.web.signup.group.templatetime true
 config set -- omero.web.signup.email.enabled true
 config set -- omero.web.signup.helpmessage 'If you have trouble creating an account please contact {{ omero_server_mail_from | default('help@example.org') }}'
-config set -- omero.web.signup.email.body 'Your login details for {{ inventory_hostname }} are:\n\nusername: {username}\npassword: {password}\n'
+config set -- omero.web.signup.email.body '{{ omero_signup_email_body }}'
```

Comment:
```suggestion
config set -- omero.web.signup.email.body {{ omero_signup_email_body }}
```

The double quoting (in this file and in the vars section) was responsible for https://github.com/ome/prod-playbooks/pull/301/files#r558310403.